ui/components/app/signature-request-siwe/signature-request-siwe-message/index.scss

```diff
@@ -0,0 +1,34 @@
+.signature-request-siwe-message {
+  flex: 1 100%;
+  border-radius: 8px;
+  padding: 8px 16px;
+  margin: 16px;
+  border: 1px solid var(--color-border-muted);
+
+  &__list {
+    flex: 1 60%;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+
+    &__item {
+      flex: 1;
+      flex-direction: column;
+
+      &__label {
+        @include H4;
+
+        margin: 8px auto;
+      }
```

Comment:
the <Typography> component also has a `boxProps` prop that can be used for margin, etc.

Reply:
Updated